monitor/views.py

```diff
@@ -89,11 +98,41 @@ def monitor_home(request):
              "sounds_analysis_failed_count": sounds_analysis_failed_count,
              "sounds_analysis_skipped_count": sounds_analysis_skipped_count,
              "gearman_status": gearman_status,
-             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count}
+             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count,
+    }
 
     return render(request, 'monitor/monitor.html', tvars)
 
 
+@cache_page(60 * 60 * 24)
+def stats_ajax(request):
+    last_week = datetime.datetime.now()-datetime.timedelta(weeks=1)
+
+    new_sounds = sounds.models.Sound.objects.filter(created__gt=last_week)\
+            .extra(select={'day': 'date(created)'}).values('day').order_by()\
+            .annotate(Count('id'))
+
+    new_users = User.objects.filter(date_joined__gt=last_week)\
+            .extra(select={'day': 'date(date_joined)'}).values('day')\
+            .order_by().annotate(Count('id'))
+
+    new_downloads = sounds.models.Download.objects.filter(created__gt=last_week)\
+            .extra({'day':"date(created)"}).values('day').order_by().annotate(Count('id'))
```

Comment:
For quotes, please use either " or ' but mixing them is a bit ugly... A good rule seems to be: always use ' except if your string has a single quote in it. I.e. 'This is a great string' but "I've got a single dash in here".